python/pyspark/sql/streaming.py

```diff
@@ -1490,6 +1588,10 @@ def _test():
     globs['df'] = \
         globs['spark'].readStream.format('text').load('python/test_support/sql/streaming')
 
+    globs['input_table_name'] = "sample_input_table_%d" % randint(0, 100000000)
+    globs['output_table_name'] = "sample_output_table_%d" % randint(0, 100000000)
+    spark.sql("CREATE TABLE %s (value string) USING parquet" % globs['input_table_name'])
```

Comment:
Can we remove these stuff, make the docstring example pretty, and add a proper unittest into `sql/tests/test_streaming.py`?

Reply:
+1